adapters/logstash/logstash.go

```diff
@@ -0,0 +1,63 @@
+package logstash
+
+import (
+	"encoding/json"
+	"errors"
+	"io"
+	"net"
+	"os"
+
+	"github.com/gliderlabs/logspout/router"
+)
+
+var hostname string
+
+func init() {
+	router.AdapterFactories.Register(NewLogstashAdapter, "logstash")
+
+	hostname, _ = os.Hostname()
```

Comment:
I see, it's a global. Maybe set this up as a field of the adapter object and do this in its constructor.